evap/evaluation/migrations/0148_evaluation_main_language.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.1.3 on 2024-11-25 20:19
+
+from django.db import migrations, models
+
+
+def _migrate(apps, schema_editor):
+    Evaluation = apps.get_model("evaluation", "Evaluation")
+    for evaluation in Evaluation.objects.filter(state__gte=40):
+        evaluation.main_language = "de"
+        evaluation.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("evaluation", "0147_unusable_password_default"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="evaluation",
+            name="main_language",
+            field=models.CharField(
+                choices=[("en", "English"), ("de", "Deutsch"), ("x", "undecided")],
```

Comment:
I want to note that now a new migration is needed when the user changes the languages in the settings. Is it okay that this is still a setting then?

If not, then this can probably become a `models.TextChoices`

Reply:
Yes, but I would do that separately from this PR